### ISSUE TITLE
Detect login failure

### ISSUE THREAD

**Issue by cbandy** [cbandy/travis-oracle#14]
Even with the fix for #10, a failed login (e.g. bad or missing password) will manifest as an error during install about a bad archive file. It would be nicer to detect this and relay an informative message to the user.

**Comment by Shoelace** [comment]
maybe add

```
[ -n "$ORACLE_LOGIN_ssousername" ] || { echo "Missing ORACLE_LOGIN_ssousername environment variable!"; exit 1; }
[ -n "$ORACLE_LOGIN_password" ] || { echo "Missing ORACLE_LOGIN_password environment variable!"; exit 1; }
```

to download.sh ?

**Comment by cbandy** [comment]
The fix for #23 improved this situation as well. When I provide an incorrect password, I now see the following:

```
2018-03-03T16:35:31.502Z ["https://login.oracle.com/oaam_server/loginAuth.do","FormSubmitted",true,true]
2018-03-03T16:35:33.892Z ["https://login.oracle.com/oaam_server/authJump.do?jump=false&clientOffset=0","Other",true,true]
2018-03-03T16:35:34.394Z ["https://login.oracle.com/oaam_server/updateLoginStatus.do","Other",true,true]
2018-03-03T16:35:34.394Z ["https://login.oracle.com/oaam_server/error.do?action=invalid_user","Other",true,true]
2018-03-03T16:35:34.395Z ["https://login.oracle.com/oaam_server/loginPage.jsp","Other",true,true]
Unable to load login result. Last response was:
Oracle
Sign in
Username
Password
 Invalid login
```
